packages/core/src/codewhisperer/commands/basicCommands.ts

```diff
@@ -263,6 +267,53 @@ export const showSsoSignIn = Commands.declare('aws.amazonq.sso', () => async ()
 // It can optionally set a customization too based on given values to match on
 
 // This command is only declared and registered in Amazon Q if Q exists
+
+export const connectCustomizationHandler =
```

Comment:
is the only difference of this implementation is that it plumbs down the client into select customization?

Reply:
yes that's right